### PR TITLE
fix: Tooltip named export

### PR DESCRIPTION
**Pull request by gyfchong**

### CODE REVIEW

**Review by christian314159 — APPROVED** [review]
Hm, I trust you here that it's necessary, but it still looks weird that we're exporting the same component once as a named export and once as the default?

**Review by HeartSquared — COMMENTED** [review]
Delete the default export please :)

**Comment by gyfchong** [comment]
Generally I think it's fine, handy in some cases, but not necessary 😄

**Review by christian314159 — APPROVED** [review]

File: draft-packages/tooltip/KaizenDraft/Tooltip/index.ts [hunk 1]
```diff
@@ -1 +1 @@
-export { default as Tooltip, TooltipProps } from "./Tooltip"
+export { Tooltip, TooltipProps } from "./Tooltip"
```
Comment (HeartSquared) [inline draft-packages/tooltip/KaizenDraft/Tooltip/index.ts:1]: We could do this, but not a big deal currently.

```suggestion
export * from "./Tooltip"
```

**Review by HeartSquared — APPROVED** [review]

**Review by christian314159 — APPROVED** [review]

**Pull request merged**